lib/std/os/bits/freebsd.zig

```diff
@@ -119,13 +119,7 @@ pub const msghdr_const = extern struct {
 pub const off_t = i64;
 pub const ino_t = u64;
 
-/// Renamed to Stat to not conflict with the stat function.
-/// atime, mtime, and ctime have functions to return `timespec`,
-/// because although this is a POSIX API, the layout and names of
-/// the structs are inconsistent across operating systems, and
-/// in C, macros are used to hide the differences. Here we use
-/// methods to accomplish this.
-pub const Stat = extern struct {
+pub const libc_stat = extern struct {
```

Comment:
Do systems other than linux need a different stat for libc vs kernel.

Reply:
No, linux is the only system where we (can) use both the libc and the kernel interface.